src/components/AddItemForm.js

```diff
@@ -0,0 +1,62 @@
+import React, {useState} from 'react';
+import fb from '../lib/firebase';
+import '../css/AddItemForm.css';
+import { v4 as uuidv4 } from 'uuid';
+
+const Form = ({token}) => {
+
+    const [itemName, setItemName] = useState("");
+    const [timeFrame, setTimeFrame] = useState(7);
+    const [lastPurchaseDate, setPurchaseDate] = useState(null);
+    const userToken = token || "userToken";
+
+    const handleSubmit = e => {
+        e.preventDefault();
+        let db = fb.firestore();
+        let data = {
+            id: uuidv4(),
+            itemName,
+            timeFrame: parseInt(timeFrame),
+            lastPurchaseDate
+        };
+        db.collection(userToken).add(data)
+        .then(() => alert(" successfully written!"))
```

Comment:
Let's update this .then to look like this. I think we should clear the form fields after sending an item. So, like this:
`.then(() => {
     alert("successfully written!");
     setItemName("");
     setTimeFrame(7);
     setPurchaseData(null);
}`
I think something like that should work, but you'd want to test before pushing. : )

Reply:
nvm...let's just add this as an issue.